app/web/.env.production

```diff
@@ -4,6 +4,4 @@ NEXT_PUBLIC_MEDIA_BASE_URL="https://user-media.couchershq.org"
 NEXT_PUBLIC_CONSOLE_BASE_URL="https://console.couchers.org"
 NEXT_PUBLIC_MAPBOX_KEY="pk.eyJ1IjoiY291Y2hlcnMiLCJhIjoiY2tpYzVnaXA0MGdmejJ4bGFrdTdrNjA0NCJ9.1VAPRbE6a_kwGw1kucppJw"
 NEXT_PUBLIC_NOMINATIM_URL="https://nominatim.openstreetmap.org/"
-NEXT_PUBLIC_TILE_URL="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
-NEXT_PUBLIC_TILE_ATTRIBUTION="Map data &copy; <a href="https://osm.org/copyright">OpenStreetMap</a> contributors"
```

Comment:
Also same, especially as this PR is so large already, would be great to do this separately.

Reply:
Moved to #4690